impl/src/main/java/org/jboss/cdi/tck/tests/se/context/ApplicationScopedObserver.java

```diff
@@ -18,10 +18,12 @@
 
 import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.enterprise.context.BeforeDestroyed;
+import jakarta.enterprise.context.Dependent;
 import jakarta.enterprise.context.Destroyed;
 import jakarta.enterprise.context.Initialized;
 import jakarta.enterprise.event.Observes;
 
+@Dependent
```

Comment:
This one is funny :laughing:

Reply:
@Ladicek right, I didn't even notice it before :-D
But it is correct as the point of it is the observation of `@Initialized(ApplicationScoped.class)`

Reply:
Agree, just had to share the fun, it's Monday morning :-)